### PR TITLE
Implement Textareas instead of inputs

### PR DESCRIPTION
**Pull request by SonicScrewdriver**
## Summary:
Swapped our usage of inputs to a textarea, and added some basic styling.

Issue: Closes Issue #5 

## Screenshots:

Screenshot 2023-10-30 at 2 56 19 PM

Screenshot 2023-10-30 at 2 56 56 PM

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -60,6 +60,24 @@ button:focus-visible {
...
+  border: none;
+  padding: 15px;
+  font-size: 20px;
+  font-weight: 400;
+  font-family: inherit;
+  resize: vertical;
+}
+textarea:focus,
+textarea:hover {
+  outline-color: #535bf2 !important;
```
Comment (nedredmond) [inline src/index.css:77]: Nice! Please set the light mode equivalent in the media query below as well

File: src/index.css [hunk 2]
```diff
@@ -60,6 +60,24 @@ button:focus-visible {
   outline: 4px auto -webkit-focus-ring-color;
 }
 
+textarea {
+  width: inherit;
+  min-height: 200px;
```
Comment (nedredmond) [inline src/index.css:65]: This is pretty big. Can we do like 2em? Also that would scale with the font size, which is dynamic and changes with viewport.

File: src/index.css [hunk 3]
```diff
@@ -60,6 +60,24 @@ button:focus-visible {
...
+  width: 50%;
+  border-radius: 5px;
+  border: none;
+  padding: 15px;
+  font-size: 20px;
```
Comment (nedredmond) [inline src/index.css:70]: Please use em (or rem) for font and padding so we scale with dynamic font size

**Comment by SonicScrewdriver** [comment]
Adding a new video since I've added functionality that allows the text area to auto-resize as necessary. The funky way scrollbars look within a textarea with border-radius + the resize handle looked pretty broken to me, so this should do away with both those issues. I wonder if "checking" a competency should scroll you down to the text area -- but that's a lower priority thought for now. :) 


https://github.com/Khan/career-competencies/assets/12463099/d4905e9c-f42b-49ba-ac43-9e7964ccf890

File: src/index.css (hunk 1 above)
Comment (nedredmond) [reply src/index.css:78]: Cool improvements! It looks like we still don't have the corresponding color in the media query.

It looks like you borrowed this color from `a`

```
a:hover {
  color: #535bf2;
}
```

and in the media query it's different

```
@media (prefers-color-scheme: light) {
  ...
  a:hover {
    color: #747bff; <<<
  }
  ...
}
```

Would you mind, for consistency?

**Review by nedredmond — APPROVED** [review]
Thanks! This looks great, but I'd like to be consistent with light/dark mode colors. See comment.

**Pull request merged**